give.php

```diff
@@ -139,6 +140,7 @@ final class Give {
 		Totals::class,
 		MigrationsServiceProvider::class,
 		RevenueServiceProvider::class,
+		MultiFormGoalsServiceProvider::class,
```

Comment:
Yeah, you're probably right. I don't personally mind having a long import list, but I'm sure this would clean that up as these won't be referenced anywhere else on the page.